disperser/batcher/batch_confirmer.go

```diff
@@ -220,6 +217,11 @@ func (b *BatchConfirmer) updateConfirmationInfo(
 		if updateConfirmationInfoErr != nil {
 			b.logger.Error("error updating blob confirmed metadata", "err", updateConfirmationInfoErr)
 			blobsToRetry = append(blobsToRetry, batchData.blobs[blobIndex])
+		} else {
+			err = b.MinibatchStore.UpdateBatchStatus(ctx, batchData.batchID, BatchStatusAttested)
```

Comment:
I think this should be outside of the for-loop, right?

Reply:
Thanks for catching this